### PR TITLE
Open Source API

### PR DESCRIPTION
**Pull request by Flemingjp**
Opensourcing the API into Github. Adding the `./Api` project and adding a initial .yml pipeline

### CODE REVIEW

**Comment by Flemingjp** [comment]
This PR also removes the `DataLoader` from the API

File: .azure/pipelines/api.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+pool:
+  vmImageName: 'windows-latest'
+
+trigger:
+  branches:
+    include:
+    - 'ApiToGithub'
```
[the remark this answers is not in the document]
Comment (beforan) [reply .azure/pipelines/api.yml:7]: for now do whatever the directory pipelines do. Probably `master` or `*`, possibly PR triggers. We may want to revisit at a later date.

**Review by beforan — CHANGES_REQUESTED** [review]

**Comment by beforan** [comment]
Do we need to a do dependency license check, similar to what we did for the directory? Or was that already done?

I think we're unlikely to hit issues since these have fewer dependencies and are .NET Core, so more likely to be MIT or similar.

**Comment by Flemingjp** [comment]
@beforan We need to check the licenses

**Comment by beforan** [comment]
> @beforan We need to check the licenses

licenses are all fine

**Review by beforan — CHANGES_REQUESTED** [review]

File: src/Api/IdentityProvider/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "iisSettings": {
```
Comment (beforan) [inline src/Api/IdentityProvider/Properties/launchSettings.json:2]: remove `iisSettings`

File: src/Api/IdentityProvider/Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:65220/",
+      "sslPort": 44382
+    }
+  },
+  "profiles": {
+    "IIS Express": {
```
Comment (beforan) [inline src/Api/IdentityProvider/Properties/launchSettings.json:11]: Remove the `IIS Express` profile.

This is a good reason for committing this file ;) to ensure everyone working on it runs it just under kestrel locally.

File: src/Api/SubmissionApi/Properties/launchSettings.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:65221/",
+      "sslPort": 44353
+    }
+  },
+  "profiles": {
+    "IIS Express": {
```
Comment (beforan) [inline src/Api/SubmissionApi/Properties/launchSettings.json:11]: as in the other project

**Review by beforan — CHANGES_REQUESTED** [review]

**Review by beforan — APPROVED** [review]

**Review by blargly — APPROVED** [review]

**Pull request merged**